dot/network/warp_sync.go

```diff
@@ -8,18 +8,34 @@ import (
 	"fmt"
 
 	"github.com/ChainSafe/gossamer/dot/network/messages"
+	"github.com/ChainSafe/gossamer/internal/primitives/consensus/grandpa"
+	"github.com/ChainSafe/gossamer/internal/primitives/core/hash"
+	"github.com/ChainSafe/gossamer/internal/primitives/runtime"
+	"github.com/ChainSafe/gossamer/internal/primitives/runtime/generic"
 	"github.com/ChainSafe/gossamer/lib/common"
 	libp2pnetwork "github.com/libp2p/go-libp2p/core/network"
 	"github.com/libp2p/go-libp2p/core/peer"
 )
 
 const MaxAllowedSameRequestPerPeer = 5
 
+type WarpSyncVerificationResult struct {
```

Comment:
would it be encoded/decoded as a P2P message?

Reply:
Nop, the p2p message is the `WarpSyncProof`, the `WarpSyncVerificationResult` is just the result of the verification function but we aren't going to send it in the network